src/app.ts

```diff
@@ -167,7 +167,8 @@ async function app() {
     }
     config = readConfig(originalInput);
     tank = fillTank(rawFSH, config);
-  } catch {
+  } catch (e) {
+    logger.error('SUSHI encountered an error while processing FSH and configuration files: ', e);
     program.outputHelp();
```

Comment:
Oh, I just re-read your opening statement and see that you said:
> However, it seems like the usage statement would be helpful in some of the cases where that try block could throw an error. 

So I guess maybe you have already identified use cases where it is helpful.  If so, feel free to ignore this!

Reply:
This line has been around for a long time and I had assumed that this was getting called when errors like incorrect paths to FSH files were used. However, when I double checked that after your question, it actually looks like we handle all the places that could happen separately now. So I think that at the point where we call `program.outputHelp()`, we have already properly handled all bad input. So I actually think that it isn't very helpful here.

Instead of outputting the help message here, I think it would be better to output the error when commander actually encounters an error. This happens if you try to use an option that we don't support. Looking at the documentation for commander, this is built in, but it is only included in a later version of commander than SUSHI was using. I updated the version to the same version that our newer repos are using, and included the function to log the help message after errors. I also removed the `outputHelp()` call from here since it does not seem useful anymore (I can't find an error that it would be helpful on).